package/batocera/core/batocera-system/Config.in

```diff
@@ -1120,6 +1120,10 @@ config BR2_PACKAGE_BATOCERA_PORTS_SYSTEMS
 	# Quake 2
 	select BR2_PACKAGE_LIBRETRO_VITAQUAKE2    if BR2_PACKAGE_BATOCERA_GLES3
 	select BR2_PACKAGE_VKQUAKE2		          if BR2_PACKAGE_BATOCERA_VULKAN && BR2_PACKAGE_XORG7
+	select BR2_PACKAGE_YQUAKE2		# ALL
+	select BR2_PACKAGE_YQUAKE2_XATRIX	# Mission Packs for yquake2
```

Comment:
ideally `if BR2_PACKAGE_YQUAKE2` so if we restrict yquake2 for any reason the other packages don't get compiled.